nautobot/extras/models/models.py

```diff
@@ -847,6 +849,16 @@ def enqueue_job(cls, func, name, obj_type, user, celery_kwargs=None, *args, sche
         if celery_kwargs is None:
             celery_kwargs = {}
 
+        job = get_job(name)
+        if job is not None:
+            try:
+                if getattr(job.Meta, "soft_time_limit", ""):
+                    celery_kwargs["soft_time_limit"] = job.Meta.soft_time_limit
+                if getattr(job.Meta, "time_limit", ""):
+                    celery_kwargs["time_limit"] = job.Meta.time_limit
+            except AttributeError:
+                pass
```

Comment:
Perhaps?

```suggestion
            if hasattr(job.Meta, "soft_time_limit"):
                celery_kwargs["soft_time_limit"] = job.Meta.soft_time_limit
            if hasattr(job.Meta, "time_limit"):
                celery_kwargs["time_limit"] = job.Meta.time_limit
```

Reply:
Ah yeah it's been a long week! How did I forget about `hasattr`?!

Reply:
Ah yeah it's been a long week! How did I forget about `hasattr`?!